tests/test_bundle_workflow.py

```diff
@@ -115,6 +119,35 @@ def test_inference_config(self, config_file):
         self._test_inferer(inferer)
         self.assertEqual(inferer.workflow_type, None)
 
+    @parameterized.expand([TEST_CASE_4])
+    def test_realtime_inference_config(self, config_file):
+        input_loader = LoadImaged(keys="image")
```

Comment:
Hi @Nic-Ma ,
   When using `LoadImage` , all the meta data is correct for the consequencial preprocessing. However, in a streaming environment, sometimes images are loading from other libraries, how could we ensure that all meta data is ready for using?

Thanks,  
Bin